wysiwyg-rails.gemspec

```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |gem|
   gem.version       = WYSIWYG::Rails::VERSION
 
   gem.add_dependency "railties", ">= 3.2", "< 5.0"
-  gem.add_dependency "font-awesome-rails", "= 4.2.0.0"
+  gem.add_dependency "font-awesome-rails", "~> 4.3.0.0"
```

Comment:
If you can change that, I will merge the commit and then you can use the gem directly from the Github branch. This way you can use Font Awesome 4.3.0.0.